GetEligibilityCheck/handler.ts

```diff
@@ -0,0 +1,98 @@
+import { Context } from "@azure/functions";
+import * as df from "durable-functions";
+import * as express from "express";
+import { fromNullable } from "fp-ts/lib/Option";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { FiscalCodeMiddleware } from "io-functions-commons/dist/src/utils/middlewares/fiscalcode";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import {
+  IResponseErrorInternal,
+  IResponseSuccessAccepted,
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseSuccessAccepted,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+import { ActivityResultSuccess } from "../EligibilityCheckActivity/handler";
+import { SottoSogliaEnum } from "../generated/definitions/ConsultazioneSogliaIndicatoreResponse";
+import { EligibilityCheck } from "../generated/definitions/EligibilityCheck";
+import { EligibilityCheckStatusEnum } from "../generated/definitions/EligibilityCheckStatus";
+import { NucleoType } from "../generated/definitions/NucleoType";
+import { initTelemetryClient } from "../utils/appinsights";
+
+type IGetEligibilityCheckHandler = (
+  context: Context,
+  fiscalCode: FiscalCode
+) => Promise<
+  | IResponseSuccessJson<EligibilityCheck>
+  | IResponseErrorInternal
+  | IResponseSuccessAccepted
+>;
+
+initTelemetryClient();
+
+function calculateBonus(familyMembers?: ReadonlyArray<NucleoType>): number {
```

Comment:
make familyMembers required and call fromNullable in the caller

Reply:
fixed in eaf20f0